scripts/get-addresses.mjs

```diff
@@ -0,0 +1,49 @@
+import { ThirdwebSDK } from "@thirdweb-dev/sdk";
+import fs from "fs";
+import path from "path";
+
+(async () => {
```

Comment:
can we add some comments on what's happening in this file in each section

replace hard coded addresses with <your-blah-here> (and add comments above them to say replace with your blah here)